src/components/ReportPage/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { PureComponent } from 'react';
+import Header from '../Header';
+import mockReport from '../../mocks/mockReport';
+import './styles.scss';
+
+class ReportPage extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      reportData: mockReport,
+    };
+  }
+
+  /* eslint-disable class-methods-use-this */
+  renderAutomationStatus(automationStatus) {
+    return (
+      automationStatus ? (
+        <span>
+          Success&nbsp;
+          <i className="fa fa-info-circle success" />
+        </span>) : (
+          <span>
+            Failed&nbsp;
+            <i className="fa fa-info-circle failed" />
+          </span>
+      )
+    );
+  }
+
+  renderTableRows() {
+    const { reportData } = this.state;
+    return reportData.map((report, index) => (
+      <tr key={report.id}>
+        <td className="numbering">{index + 1}</td>
+        <td className="column1">{report.date}</td>
+        <td>{report.fellowName}</td>
+        <td>{report.partnerName}</td>
+        <td>{report.type}</td>
+        <td>{this.renderAutomationStatus(report.slackAutomation.success)}</td>
+        <td>{this.renderAutomationStatus(report.emailAutomation.success)}</td>
+        <td>{this.renderAutomationStatus(report.freckleAutomation.success)}</td>
+      </tr>
+    ));
+  }
+
+
+  render() {
+    return (
+      <div>
+        <Header />
```

Comment:
Did we settle on having a single page? Or should we have in mind future scalability (additional pages such as one with visual representation)? If so I think it is not a good idea to place the Header component here.

Reply:
For now, we have settled on having a single page.
About future scalability of having additional pages, we can't really predict what the UI would become as at then so I think our component composition should  be based on what we currently know(or can predict).
If there is a need to move the header in future cases, it would definately be moved 🙂.

Reply:
I think it is best to build something that is easily scalable from the beginning

Reply:
Where do Where do you suggest I move it?

Reply:
The page with routes

Reply:
This conversation was resolved in a sync with the TTL for this project(@tadesanya).
The resolution was to make use of the Header component only when it is needed, consequently, any component(page) that requires the Header component should require it as opposed to having it in the routes.
cc: @Betty-Kebenei